### DISCUSSION TITLE
Lab6 FPGA上模擬出現亂碼

### DISCUSSION THREAD

**Discussion by Alex17898** [bol-edu/HLS-SOC-Discussions#162] [Q&A] [answered]
想請問一下在firmware模擬時是傳送一個TEST的字串，模擬也成功，但到了FPGA卻出現亂碼，想了解一下這是甚麼原因造成的，謝謝!
![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/134195336/ecd6174c-6def-4cdf-aced-bc7d3fd20e45)
![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/134195336/3e919f52-a91b-4627-aa4b-2186aa174ce8)

**Comment by LingChiYang** [comment, marked as answer]
同學好

會出現亂碼是因為收回來的資料錯誤，可以請同學先確認模擬時tx rx是不是同一筆資料

**Reply by Alex17898** [reply]
後來有發現不是同一筆資料是因為有不小心修改到mprj_io[6]，才會造成錯誤，目前已解決，謝謝助教!
